015/master

neugierige opened the pull request:
addressing 2 things from issue #17: 1) type in the instructions, and 2) clarify Checkpoint 1 and update code to make it run as expected

Commit:
fixed typo in Checkpoint 0 instruction

as per issue #17

Commit:
replaced function "declaration" to "expression"

as part of the solution for issue #17, as per conversation
with @gaand on 12/8/16

Commit:
function declaration changed

as part of solution for issue #17, as per conversation
with @gaand on 12/8/16

gaand commented on study.md:
```diff
@@ -112,26 +112,27 @@ var myModule = (function(arg, transform) {
 ### Checkpoint 1
 
 1.  Open [`lib/inline.js`](lib/inline.js).
 1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
```
Maybe say:

```diff
-1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
+1.  Put a pair of parens (`(` and `)`) around the function assigned to `f`.

gaand commented on study.md:
```diff
@@ -112,26 +112,27 @@ var myModule = (function(arg, transform) {
 1.  Open [`lib/inline.js`](lib/inline.js).
 1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
+    Assign this function to a new variable, and update the assignment of
+    `ourModule` to equal this new variable.
```
This addition seems unnecessary so I suggest removing it.

neugierige replied:
Agreed. That was my original proposed change, I didn't intend to include it in the commit.

gaand reviewed (commented):
A couple of notes.

Commit:
updated 2nd item in Checkpoint 1 instructions

To reflect updated function expression in lib/inline.js
As part of a solution for issue #17
